4-fleetscope/modules/env_baseline/acm.tf

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  fleet_membership_re = "//gkehub.googleapis.com/(.*)$"
+}
+
+resource "google_project_service" "acm" {
+  project = var.project_id
+  service = "anthosconfigmanagement.googleapis.com"
```

Comment:
Would you prefer to enable when the project is created?

https://github.com/GoogleCloudPlatform/terraform-google-enterprise-application/blob/main/2-multitenant/modules/env_baseline/main.tf#L60

Reply:
yes, moved to activation at project creation time